### ISSUE TITLE
[Bug] Image isn't displayed

### ISSUE THREAD

**Issue by Youssef1313** [xamarin/Xamarin.Forms#8623]
## Description

I'm trying to display an image from a URL, but it seems that it doesn't work for me. (Not sure if it's a bug or I'm missing something)

### Steps to Reproduce

Using the following xaml:

```xaml
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
             mc:Ignorable="d"
             x:Class="App1.MainPage">

    <StackLayout>
        <Label Text="HelloWorld"></Label>
        <Image Source="https://placehold.it/100"></Image>
    </StackLayout>

</ContentPage>
```

### Expected Behavior

The image displays under the label.

### Actual Behavior

Only the label was displayed.

### Basic Information

- Version with issue: Xamarin.Forms 4.2.0.709249
- Last known good version: Don't know, only used this version (starting learning Xamarin).
- IDE: Visual Studio 2019
- Platform Target Frameworks:
  - iOS:  Haven't tested on iOS (I don't have a Mac currently).
  - Android: Android 9.0 (Pie) 
### Screenshots

![image](https://user-images.githubusercontent.com/31348972/69435647-9d888e00-0d48-11ea-972e-95e105d4c8aa.png)

**Comment by jfversluis** [comment]
Hey @Youssef1313 thanks for the report!

I've tried to reproduce it, but I seem unable to. I've taken your exact XAML and it simply just works.
Could you try the latest Xamarin.Forms packages?

![image](https://user-images.githubusercontent.com/939291/69436845-76c75900-0d42-11ea-86b2-3b4d2851c3ba.png)

I've added a reproduction project here, maybe you could try that one? Thanks!

[Repro8623.zip](https://github.com/xamarin/Xamarin.Forms/files/3880121/Repro8623.zip)

**Comment by Youssef1313** [comment]
@jfversluis, Thanks for the fast response :)
Unfortunately, I don't know why I can't run the repro project.
Visual Studio seems to be stuck at "Runtime checks completed"

![image](https://user-images.githubusercontent.com/31348972/69438570-f1e23c80-0d4d-11ea-8046-03726dd01f32.png)

**Comment by Youssef1313** [comment]
@jfversluis, Thanks for your response. Unfortunately, I couldn't run your repro project. It doesn't build and there is that hello triangle in the dependencies in solution explorer.
NuGet Package Manager seems to be stuck at "Loading" in both, your project and my project.

![image](https://user-images.githubusercontent.com/31348972/69439629-06bfcf80-0d50-11ea-8742-75793dae9788.png)

**Comment by jfversluis** [comment]
Is there any output in the consoles? What dependencies are giving you warnings and with what message?

**Comment by Youssef1313** [comment]
All the dependencies in your project were giving warnings. I've restarted Visual Studio and the warnings gone somehow, and also your repro project worked. But I can't get my project to work. I've pushed my project to [this repo](https://github.com/Youssef1313/Xamarin.Forms8623).

**Comment by jfversluis** [comment]
I’m not behind a computer right now, but I see that your project is using Xamarin.Forms 4.2. Could you update your project to 4.3? I suspect that your problem might be fixed in a newer version :)

**Comment by Youssef1313** [comment]
@jfversluis, That fixed it. :)
Was this a known bug in 4.2 that got fixed in 4.3 ?

**Comment by jfversluis** [comment]
I think it was. Sorry about that!

Hope you can make some good progress on starting with Forms now, good luck! If we can help you with anything else, please let us know
